File: GrowthWare_VB/GrowthWareFramework/ModelObjects/MAccountProfileCollection.vb
Imports GrowthWare.Framework.ModelObjects.Base
Imports System.Runtime.Serialization

Namespace ModelObjects
    <Serializable(), CLSCompliant(True)> _
    Public NotInheritable Class MAccountProfileCollection
		Inherits MProfileCollection

		Protected Sub New(ByVal info As SerializationInfo, ByVal context As StreamingContext)
			MyBase.New(info, context)
		End Sub
	End Class
End Namespace

File: GrowthWare_CS/GrowthWareFramework/ModelObjects/MAccountProfileCollection.cs
using System;
using System.Runtime.Serialization;
using GrowthWare.Framework.ModelObjects.Base;

namespace GrowthWare.Framework.ModelObjects
{
	[Serializable(), CLSCompliant(true)]
	public sealed class MAccountProfileCollection : MProfileCollection
	{

		MAccountProfileCollection() : base()
		{
			// do nothing
		}
		MAccountProfileCollection(SerializationInfo info, StreamingContext context) : base(info, context)
		{
			// do nothing
		}
	}
}
